pkgs/by-name/si/sing-box/package.nix

```diff
@@ -50,6 +49,10 @@ buildGoModule (finalAttrs: {
       --replace-fail "/usr/bin/sing-box" "$out/bin/sing-box" \
       --replace-fail "/bin/kill" "${coreutils}/bin/kill"
     install -Dm444 -t "$out/lib/systemd/system/" release/config/sing-box{,@}.service
+
+    install -Dm444 release/config/sing-box.sysusers $out/lib/sysusers.d/sing-box.conf
+    install -Dm444 release/config/sing-box.rules $out/share/polkit-1/rules.d/sing-box.rules
```

Comment:
These rules only works if sing-box is running under the sing-box user, maybe we shouldn't allow users to customize the user/group name.

Reply:
I think we can just have the contents in security.polkit.extraConfig.

Reply:
Or we could modify the file to have the policy scoped to $out/bin/sing-box.  Not sure if that's possible though.